compiler/test/unit/semantic/imports.d

```diff
@@ -14,6 +14,7 @@ import support;
 unittest
 {
     initDMD();
+    scope(exit) deinitializeDMD();
```

Comment:
for consistency with other files (e.g. https://github.com/dlang/dmd/blob/master/compiler/test/unit/frontend.d ) please do 
```d
import support : afterEach;

@afterEach deinitializeFrontend()
{
    import dmd.frontend : deinitializeDMD;
    deinitializeDMD();
}
```